src/main/java/frc/robot/Constants.java

```diff
@@ -127,7 +127,11 @@ public static class kManipulator {
     public static final double kD = 0;
     public static final double kF = 0;
 
-    public static final double kG = 1;
+    public static final double kG_UP = 1;
+    public static final double kG_DOWN = 1;
+
+    public static final double kP_UP = 0; //set this
+    public static final double kP_DOWN = 0; //set this
 
     public static final int kSpinMotorID = 24;
```

Comment:
spin motors shoudnt have k infrom of them should be SPIN_MOTOR, also should be in the kPorts class